README.md

```diff
@@ -27,6 +27,12 @@ This quickstart project comes with the following by default:
 ## Quickstart
 
 Follow this section to get this project working. Before you begin, ensure you have the latest version of hasura cli tool installed.
+Make sure that you are logged in before cloning the project:
```

Comment:
Hey, this will be included in the "Clone & Deploy" section of Hasura hub and is not needed here.

Reply:
Oh I see. Will fix it.